LayoutTests/imported/w3c/web-platform-tests/css/selectors/popover-open-with-has-sibling-selector-expected.xht

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+ <head>
+  <title>CSS Reftest Reference</title>
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/" />
+  <style type="text/css"><![CDATA[
+  div
+  {
+  background-color: green;
```

Comment:
What's up with this indentation!?

Reply:
This is just copied from `css/reference/ref-filled-green-100px-square.xht` which is like this upstream. The `-expected.xht` file clones for the WebKit repo should match the actual references upstream.